catalog-info-1.yaml

```diff
@@ -12,7 +12,7 @@ metadata:
     backstage.io/techdocs-ref: dir:./
 spec:
   type: service
-  owner: david@roadie.io
+  owner: dtuite
```

Comment:
doesn't this need to be `user:dtuite` or is this on purpose?

Reply:
I suspect it does need to be user:dtuite but I'm trying to test what changes in the UI with the various different settings.